MoltenVK/MoltenVK/GPUObjects/MVKDevice.mm

```diff
@@ -2480,7 +2494,7 @@ static uint32_t mvkGetEntryProperty(io_registry_entry_t entry, CFStringRef prope
 	if (supportsMTLGPUFamily(Apple3)) { mtlFam = MTLGPUFamilyApple3; }
 	if (supportsMTLGPUFamily(Apple4)) { mtlFam = MTLGPUFamilyApple4; }
 	if (supportsMTLGPUFamily(Apple5)) { mtlFam = MTLGPUFamilyApple5; }
-#if MVK_IOS || MVK_MACOS_APPLE_SILICON
+#if MVK_XCODE_12
 	if (supportsMTLGPUFamily(Apple6)) { mtlFam = MTLGPUFamilyApple6; }
 	if (supportsMTLGPUFamily(Apple7)) { mtlFam = MTLGPUFamilyApple7; }
```

Comment:
This is part of what's breaking tvOS.

Reply:
Thanks. Yeah. Working on the various combinations of platforms and Xcode versions. We can now add Apple8 on iOS too I see.

Will submit an update soon.

Reply:
Fixed.